src/main/java/uk/gov/hmcts/reform/blobrouter/config/ServiceConfiguration.java

```diff
@@ -32,4 +44,8 @@ public void setStorageConfig(List<StorageConfigItem> storageConfigItems) {
             .map(StorageConfigItem::getSourceContainer)
             .collect(toList());
     }
+
+    public List<String> getSourceContainers() {
```

Comment:
why not use existing `getEnabledSourceContainers`?

Reply:
we may need to get for not enabled ones?

Reply:
Probably not.
If `StaleBlobFinder` is used for alerting, we don't want to be notified about files in disabled containers probably.

Reply:
it is going to be used manually?